### PR TITLE
localizable menu multimedia

### PR DESCRIPTION
**Pull request by wpride**

### CODE REVIEW

**Comment by wpride** [comment]
cross with https://github.com/dimagi/commcare/pull/71

@ctsims

File: app/src/org/commcare/android/adapters/MenuAdapter.java [hunk 1]
```diff
@@ -130,24 +130,19 @@ public MenuAdapter(Context context, CommCarePlatform platform, String menuID){
...
         }
         
-        objectData = new Object[items.size()];
-        items.copyInto(objectData);
-    }
-    
-    public String getMenuTitle() {
-        return menuTitle;
+        displayableData = new Displayable[items.size()];
+        items.copyInto(displayableData);
```
Comment (orangejenny) [inline app/src/org/commcare/android/adapters/MenuAdapter.java]: Might as well make items `Vector<Displayable>` when it's created.

File: app/src/org/commcare/android/adapters/MenuAdapter.java [hunk 2]
```diff
@@ -234,46 +229,19 @@ else if(datum.getNodeset() == null) {
         if(mQuestionText != null) {
             mQuestionText = Localizer.processArguments(mQuestionText, new String[] {""}).trim();
         }
-        emv.setAVT(mQuestionText, getAudioURI(mObject), getImageURI(mObject), iconChoice);
+        emv.setAVT(mQuestionText, mObject.getAudioURI(), mObject.getImageURI(), iconChoice);
         return emv;
     }
-    
-    /*
-     * Helpers to make the getView call Entry/Menu agnostic
```
Comment (orangejenny) [inline app/src/org/commcare/android/adapters/MenuAdapter.java]: nice job replacing these with the interface - much cleaner

File: app/src/org/commcare/android/view/GridMediaView.java [hunk 3]
```diff
@@ -48,7 +49,8 @@ public GridMediaView(Context c, EvaluationContext ec) {
 
 
     public void setDisplay(DisplayUnit display) {
-        setAVT(Localizer.processArguments(display.getText().evaluate(ec), new String[] {""}).trim(), display.getImageURI());
+        DisplayData mData = display.evaluate();
```
Comment (orangejenny) [inline app/src/org/commcare/android/view/GridMediaView.java]: Why don't you pass ec here?

**Comment by orangejenny** [comment]
minor comments only, waiting on the commcare PR before merging this one

**Comment by wpride** [comment]
@orangejenny should be gtg

**Pull request merged**